recipes/gtest/all/conandata.yml

```diff
@@ -12,6 +12,5 @@ patches:
   "1.10.0":
     - patch_file: "patches/gtest-1.10.0.patch"
       base_path: "source_subfolder"
-  "1.10.0":
```

Comment:
this is interesting typo, @uilianries do you think if it's something that can be easily validated by hook?

Reply:
@SSE4 Yes, I think so.